starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,116 @@
-// Names and Input
-
+//Names and input
+var hacker1 = "Bob";
+// var hacker2 = prompt("What is the navigator's name?");
+var hacker2 = "Stephen";
```

Comment:
We suppose to do something like this:

var hacker2 = prompt("What's navigator's name?");
console.log("The navigator's name is " + hacker2 + ".");